src/univariate/continuous/noncentralchisq.jl

```diff
@@ -1,3 +1,24 @@
+"""
+    NoncentralChisq(ν, λ)
+
+The *noncentral chi-squared distribution* with `ν` degrees of freedom and noncentrality parameter `λ` has the probability density function
+
+$f(x; ν, λ) = \frac{1}{2}e^{-(x + λ)/2}\frac{x}{λ}^{ν/4-1/2}I_{ν/2-1}(\sqrt{λx}) \quad x>0.$
```

Comment:
This might render better as
```tex
f(x; \nu, \lambda) = \frac{1}{2} e^{-(x + \lambda)/2} \left( \frac{x}{\lambda} \right)^{\nu/4-1/2} I_{\nu/2-1}(\sqrt{\lambda x}), \quad x > 0
```
Typically TeX prefers getting its recognized sequences, e.g. `\lambda`, rather than raw Unicode, e.g. λ. Whether raw Unicode works at all often depends on the particular TeX distribution and/or whether `inputenc` is active.

Also note the parentheses around the second fraction. What I've shown above should exactly match what's on Wikipedia, with the exception of `\nu` as the noncentrality parameter instead of `k`.